src/content/en/2021/resource-hints.md

```diff
@@ -266,18 +266,18 @@ The `crossorigin` attribute is used to indicate whether CORS must be used when f
     <tbody>
       <tr>
         <td>not set</td>
-        <td class="numeric">66.61%</td>
-        <td class="numeric">65.91%</td>
+        <td class="numeric">66.6%</td>
+        <td class="numeric">65.9%</td>
       </tr>
       <tr>
         <td><code>crossorigin</code> (or equivalent)</td>
-        <td class="numeric">14.50%</td>
-        <td class="numeric">13.48%</td>
+        <td class="numeric">14.5%</td>
+        <td class="numeric">13.5%</td>
       </tr>
       <tr>
         <td><code>use-credentials</code></td>
-        <td class="numeric">0.04%</td>
-        <td class="numeric">0.04%</td>
+        <td class="numeric">0.1%</td>
```

Comment:
@tunetheweb is this intentionally rounding up?

Reply:
Good spot. What do you think about `< 0.1%` to show it is used and not 0?